pkg/registration/hub/manager.go

```diff
@@ -159,13 +163,13 @@ func (m *HubManagerOptions) RunControllerManagerWithInformers(
 	for _, enabledRegistrationDriver := range m.EnabledRegistrationDrivers {
 		switch enabledRegistrationDriver {
 		case "csr":
-			csrDriver, err := csr.NewCSRHubDriver(kubeClient, kubeInformers, m.ClusterAutoApprovalUsers, controllerContext.EventRecorder)
+			csrDriver, err := csr.NewCSRHubDriver(kubeClient, kubeInformers, m.ClusterAutoApprovalUsers, m.AutoApprovedCSRUsers, controllerContext.EventRecorder)
```

Comment:
I think we should not introduce a new parameter `autoApprovedCSRUsers`, we may rename the  `clusterAutoApprovalUsers ` to `autoApprovedCSRUsers ` and  handle the compatibility here, as long as `autoApprovedCSRUsers` is set, we will use it, else, use the `clusterAutoApprovalUsers`

Reply:
addressed